File: MyMindNotes/app/src/main/java/com/android/mymindnotes/retrofit/GetUserInfoApi.java
package com.android.mymindnotes.retrofit;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface GetUserInfoApi {
    @GET("/api/member/getUserInfo/{user_index}")
    Call<Map<String, Object>> getUserInfo(@Path("user_index") int user_index);
}

File: MyMindNotes/data/src/main/java/com/android/mymindnotes/data/retrofit/api/user/GetUserInfoApi.kt
package com.android.mymindnotes.data.retrofit.api.user

import com.android.mymindnotes.core.dto.GetUserInfoResponse
import retrofit2.http.GET
import retrofit2.http.Path

interface GetUserInfoApi {
    @GET("/api/member/getUserInfo/{user_index}")
    suspend fun getUserInfo(
        @Path("user_index") user_index: Int
    ): GetUserInfoResponse
}